seedlet-blocks/experimental-theme.json

```
@@ -0,0 +1,58 @@
+{
+	"global": {
+		"presets": {
+			"font-size": [
+				{
+					"slug": "tiny",
+					"value": "14px"
+				},
+				{
+					"slug": "small",
+					"value": "16px"
+				},
+				{
+					"slug": "medium",
+					"value": "90px"
```

Did you mean for this to be `90px`? 😄 

```suggestion
					"value": "18px"
```